Documentation/Sprint 2/reflection.tex

```diff
@@ -0,0 +1,218 @@
+\item Install a PostgreSQL instance on a vps.
+\end{enumerate}
+
+& 
+\begin{enumerate}
+\item Jan-Willem Gmelig Meyling
+\item Jaap Heijligers
+\end{enumerate}
+
+& 
+\begin{enumerate}
+\item 2 hours
+\item 2 hours
+\end{enumerate}
+
+&
+\begin{enumerate}
+\item 1 hour
+\item 2 hours
+\end{enumerate}
+
+&
+\begin{enumerate}
+\item Yes
+\item Yes
+\end{enumerate}
+
+&
+\begin{enumerate}
+\item 
+\item Had some access problems from the outside.
+\end{enumerate}
+\\
+
+As a developer I want to make an algorithm that takes low-level music features and produces an valence-arousal vector.
+&
+\begin{enumerate}
+\item Make a predictor implementation using the recommendation-library for the valence/arousal vector.
+\item Make a test dataset with songs and our personal preferences to test our algorithm.
+\item Test the predictor comparing our preference with the predicted preference.
+\item Tweak the algorithm in order to produce reasonable vector-predictions.
+\end{enumerate}
+
+&
+\begin{enumerate}
+\item Tim van der Lippe
+\item Eva Anker
+\item Tim van der Lippe
+\item Eva Anker
+\end{enumerate}
+
+&
+
+\begin{enumerate}
+\item 2 hours
+\item 6 hours
+\item 2 hours
+\item 10 hours
+\end{enumerate}
+
+&
+\begin{enumerate}
+\item 4 hours
+\item 4 hours
+\item 2 hours
+\item 3 hours
+\end{enumerate}
+
+&
+\begin{enumerate}
+\item Yes
+\item Yes
+\item Yes
+\item No
+\end{enumerate}
+&
+\begin{enumerate}
+\item Took some extra time due to fitting the library to our needs.
+\item 
+\item Jan-Willem took over this issue.
+\item Not finished due to an unforseen amount of documentation work. Has been transferred to next sprint.
+\end{enumerate}
+
+\end{tabular}
+\end{table}
+\begin{table}[h]
+\begin{tabular}{|p{3cm}|p{5.0cm}|p{3.0cm}|p{2.2cm}|p{2.2cm}|p{2.0cm}|p{5cm}}
+As a user I want to go to a room and chat whit the people that are there with me.
+I want to be able to save the url from a room and come back later.
+&
+\begin{enumerate}
+\item Make the API on the backend for the chats.
+\item Make the API on the backend for the rooms.
+\item Let the frontend do API calls to the backend to receive room and chat data.
+\end{enumerate}
+
+&
+
+\begin{enumerate}
+\item Jaap Heijligers
+\item Tim van der Lippe
+\item Gijs Weterings
+\end{enumerate}
+
+&
+\begin{enumerate}
+\item 6 hours
+\item 6 hours
+\item 8 hours
+\end{enumerate}
+
+&
+\begin{enumerate}
+\item 3 hours
+\item 6 hours
+\item 9 hours
+\end{enumerate}
+
+&
+\begin{enumerate}
+\item Yes
+\item Yes
+\item Yes
+\end{enumerate}
+&
+\begin{enumerate}
+\item Swapped this issue and the next. Tim made this one. We found the communication between the frontend and backend tasks a bit troublesome, as we did not agree on a specific standard until later in the sprint.
+\item Swapped this issue and the next. Jaap made this one. We found the communication between the frontend and backend tasks a bit troublesome, as we did not agree on a specific standard until later in the sprint.
```

Comment:
'the next' should be 'the former'. The last sentence is double and could be removed.